### ISSUE TITLE
Add `--always-on-top=topmost/overlay` flag making app visible even over macOS fullscreen apps

### ISSUE THREAD

**Issue by 12-Seconds** [jiahaog/nativefier#1038]
**Homework**

- [x] I looked at `nativefier --help` and https://github.com/jiahaog/nativefier/blob/master/docs/api.md , no existing option fits my needs.
- [x] I searched existing issues, open & closed. Yes, my feature request is new.
- [x] I'm using the latest version available at https://github.com/jiahaog/nativefier/releases

**Problem statement**

Nativefier should include a feature to allow it to overlay full screen apps on macOS.

**Suggested solution**

An option could be added, such as --overlay, to allow the app to be visible over full screen apps.

**Context**

 - Nativefier: (10.1.0)
 - Node.js: (14.9.0)
 - Npm: (6.14.7)
 - OS: (macOS 10.15.6)

**Comment by ronjouch** [comment]
> to allow the app to be visible over full screen apps

@12-Seconds did you try [flag `--always-on-top`](https://github.com/jiahaog/nativefier/blob/master/docs/api.md#always-on-top) ?

**Comment by 12-Seconds** [comment]
@ronjouch Yes, I've tried that flag, but it does not work on full screen apps, only windowed apps.

**Comment by ronjouch** [comment]
> @ronjouch Yes, I've tried that flag, but it does not work on full screen apps, only windowed apps.

@12-Seconds do other Electron apps do that? For example, when you make a Slack/Skype call, does the small call indicator (black rectangle, showing the name of the person being called, a timer, sometimes a hang-up button, sometimes the video of the person being called) stay above full screen apps?

**Comment by 12-Seconds** [comment]
@ronjouch no, I haven't seen any electron apps do this

**Comment by ronjouch** [comment]
> no, I haven't seen any electron apps do this

@12-Seconds yeah, doubt this is possible, then. With that, closing, and:

1. Consider requesting the feature in https://github.com/electron/electron/issues/
2. If the feature lands or if we're wrong and it's feasible with current Electron, do chime in here and I'll re-open the issue.

**Comment by 12-Seconds** [comment]
@ronjouch came across [this](https://github.com/electron/electron/issues/10078) issue in the electron repo. Over in [this](https://github.com/kamranahmedse/pennywise/issues/5) issue it's been said that it was fixed. Would it be possible to add this to Nativefier?

**Comment by ronjouch** [comment]
Re-opening, as:

1. https://github.com/electron/electron/issues/10078#issuecomment-425690924 offers a workaround.
2. https://github.com/electron/electron/issues/10078#issuecomment-438118254 says *"this is now in master"*, although with a sloppy dead link and no link to any commit/docs 😕. To Nativefier passersby interested in implementing this issue, please prefer digging what this comment meant (looking at Electron commits shortly before this comment would be a good start) and use it, which is probably preferable as more reliable.

**Comment by 12-Seconds** [comment]
@ronjouch I found the following:

https://github.com/electron/electron/pull/11599
https://github.com/electron/electron/pull/14122

However, it seems that it isn't possible to overlay full screen apps anymore, starting with macOS 10.14, as indicated by [pull request #21706](https://github.com/electron/electron/pull/21706)
